credsweeper/secret/config.json

```diff
@@ -1,150 +1,1097 @@
 {
-    "exclude": {
-        "pattern": [],
-        "extension": [
-            ".7z",
-            ".aac",
-            ".aar",
-            ".avi",
```

Comment:
In my VS Code 4 spaces is default value of formatting..
Are you using Pycharm?

Reply:
Yes. it was default in Pycharm. I replaced to 4 spaces.